### PR TITLE
Update gke-install.md

### PR DESCRIPTION
**Pull request by pietrangelo**
This simplify the installation of Entando on GKE if a user wants to use a storage class which support RWX(Read Write Many) mode.

### CODE REVIEW

File: vuepress/docs/next/tutorials/getting-started/gke-install.md [hunk 1]
```diff
@@ -146,31 +146,13 @@ You do not need clustered storage to scale an Entando Application if you schedul
 :::
 
 ### Clustered Storage Using GCP Cloud Filestore
-1. In the left menu of the GCP portal, find the Storage section and select `Filestore` -> `Instances`
+1. In the cluster `DETAILS` tab, under `Features` find the `Filestore CSI driver` property and enable it.
```
Comment (jyunmitch) [inline vuepress/docs/next/tutorials/getting-started/gke-install.md:149]:
```suggestion
1. In the cluster `DETAILS` tab, under `Features`, find the `Filestore CSI driver` property and enable it
```

File: vuepress/docs/next/tutorials/getting-started/gke-install.md [hunk 2]
```diff
@@ -146,31 +146,13 @@ You do not need clustered storage to scale an Entando Application if you schedul
...
 
-2. Enable the Filestore, if you haven't already
+2. Wait a couple of minutes for the changes to propagate
```
Comment (jyunmitch) [inline vuepress/docs/next/tutorials/getting-started/gke-install.md:151]:
```suggestion
2. Wait for the changes to propagate
```

File: vuepress/docs/next/tutorials/getting-started/gke-install.md [hunk 3]
```diff
@@ -146,31 +146,13 @@ You do not need clustered storage to scale an Entando Application if you schedul
...
 
-3. Select `Create Instance`
+3. Ensure also the `Cloud Filestore API` is enabled under `API and services` on the main left menu.
```
Comment (jyunmitch) [inline vuepress/docs/next/tutorials/getting-started/gke-install.md:153]:
```suggestion
3. Ensure that the `Cloud Filestore API` is enabled under `API and services` on the main left menu
```

File: vuepress/docs/next/tutorials/getting-started/gke-install.md [hunk 4]
```diff
@@ -146,31 +146,13 @@ You do not need clustered storage to scale an Entando Application if you schedul
...
-  helm install nfs-subdir-external-provisioner nfs-subdir-external-provisioner/nfs-subdir-external-provisioner \
-      --set nfs.server=YOUR-NFS-IP \
-      --set nfs.path=YOUR-NFS-PATH
-  ```
-
-  Learn about the provisioner and additional configuration options at
-  <https://github.com/kubernetes-sigs/nfs-subdir-external-provisioner>
-
-7. Verify that your client provisioned successfully. This is indicated by the presence of the storage class `nfs-client` in the output of the following command.
+3. Verify that your client provisioned successfully. This is indicated by the presence of the storage class with the `rwx` suffix in the output of the following command.
```
Comment (jyunmitch) [inline vuepress/docs/next/tutorials/getting-started/gke-install.md:155]:
```suggestion
4. Verify that your client was provisioned successfully. This is indicated by the presence of the storage class with `rwx` suffix in the output following the command.
```

File: vuepress/docs/next/tutorials/getting-started/gke-install.md [hunk 5]
```diff
@@ -179,15 +161,19 @@ Example output:
...
+enterprise-multishare-rwx   filestore.csi.storage.gke.io   Delete          WaitForFirstConsumer   true                   67m
+enterprise-rwx              filestore.csi.storage.gke.io   Delete          WaitForFirstConsumer   true                   67m
+premium-rwo                 pd.csi.storage.gke.io          Delete          WaitForFirstConsumer   true                   116m
+premium-rwx                 filestore.csi.storage.gke.io   Delete          WaitForFirstConsumer   true                   67m
+standard                    kubernetes.io/gce-pd           Delete          Immediate              true                   116m
+standard-rwo (default)      pd.csi.storage.gke.io          Delete          WaitForFirstConsumer   true                   116m
+standard-rwx                filestore.csi.storage.gke.io   Delete          WaitForFirstConsumer   true                   67m
 ```
 
 8. Add the variables below to your operator `ConfigMap`
```
Comment (jyunmitch) [inline vuepress/docs/next/tutorials/getting-started/gke-install.md:174]:
```suggestion
5. Add the variables below to your operator `ConfigMap`
```

File: vuepress/docs/next/tutorials/getting-started/gke-install.md [hunk 6]
```diff
@@ -179,15 +161,19 @@ Example output:
...
 ```
-entando.k8s.operator.default.clustered.storage.class: "nfs-client"
+entando.k8s.operator.default.clustered.storage.class: "standard-rwx"
 entando.k8s.operator.default.non.clustered.storage.class: "standard"
 ```
```
Comment (jyunmitch) [inline vuepress/docs/next/tutorials/getting-started/gke-install.md:179]: the number 9 in line 180 should be '6'

File: vuepress/docs/next/tutorials/getting-started/gke-install.md (hunk 4 above)
Comment (jyunmitch) [reply vuepress/docs/next/tutorials/getting-started/gke-install.md:155]: unsure of what was meant by 'the following command'... is the presence of the storage class with the xxx suffix supposed to follow the command? that's what my update suggests

Comment (pietrangelo) [reply vuepress/docs/next/tutorials/getting-started/gke-install.md:155]: by `the following command` I mean `kubectl get sc` which returns the list of the new generated storage classes

**Review by jyunmitch — APPROVED** [review]

**Pull request merged**